instrumentation/nethttp/client.go

```diff
@@ -1,18 +1,18 @@
 package nethttp
 
 import (
+	"bytes"
 	"context"
+	"github.com/opentracing/opentracing-go"
```

Comment:
Make sure you have the IDE configured to separate standard library imports from third party imports

Reply:
Don't know how to do that in the IDE, but sorted manually.